### PR TITLE
Project Complete with functionality

### PR DESCRIPTION
**Pull request by rutulsheladiya31**
create a form and store data on localstorage using javascript.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+                </div>
+                <hr>
+
+                <div class="main-thd-div">
+                    <div class="radio-btn">
+                        <label>Gender:</label>
+                        <input type="radio" id="gender" name="gender" value="Male">Male
+                        <input type="radio" id="gender" name="gender" value="Female">Female
+                    </div>
+                    <!-- <div class="checkbox-btn">
```
Comment (kerulshahsimform) [inline index.html:129]: Please remove these unwanted / commented codes.

Comment (rutulsheladiya31) [reply index.html:129]: Done.

File: showdata.html [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+            <td id="sectelephone"></td>
+        </tr>
+        <tr>
+            <th>Relation</th>
+            <td id="relation"></td>
+        </tr>
+    </table>
+    <div class="clear-btn">
+        <button type="submit" onclick="cleardata()">clear</button>
+    </div>
```
[the remark this answers is not in the document]
Comment (rutulsheladiya31) [reply showdata.html:109]: Done

File: js/fetch-data.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+function cleardata() {
+    localStorage.clear();
+    console.log("clear");
+    location.reload();
+}
+
+
+{
+    /* <script>
```
[the remark this answers is not in the document]
Comment (rutulsheladiya31) [reply js/fetch-data.js:55]: Done.

File: js/script.js [hunk 4]
```diff
@@ -0,0 +1,307 @@
...
+
+            // validation for residency
+            if (document.getElementById('residency').value == "") {
+                alert("Enter Residency");
+                document.getElementById('residency').focus();
+                return false;
+            }
+
+            // validation for mobile number
+            var Mobile = document.getElementById('fstmobileno').value;
```
[the remark this answers is not in the document]
Comment (rutulsheladiya31) [reply js/script.js]: Done.

File: js/script.js [hunk 5]
```diff
@@ -0,0 +1,306 @@
...
+    // validation for residency
+    if (document.getElementById('residency').value == "") {
+        alert("Enter Residency");
+        document.getElementById('residency').focus();
+        return false;
+    }
+
+    // validation for mobile number
+    var Mobile = document.getElementById('fstmobileno').value;
+    if (Mobile == "") {
```
[the remark this answers is not in the document]
Comment (rutulsheladiya31) [reply js/script.js]: Done.

**Pull request merged**